addons/website/tests/test_ir_asset.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestIrAsset(odoo.tests.HttpCase):
+
+    def test_01_website_specific_assets(self):
+        IrAsset = self.env['ir.asset']
+        Website = self.env['website']
+
+        website_1 = Website.create({'name': 'Website 1'})
+        website_2 = Website.create({'name': 'Website 2'})
```

Comment:
Test working as expected :+1: 
Juste can you put `Website 1` and `Website 2` in double quotes as they are  human readable strings. ty